.github/workflows/benchmarks.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         dependencies:
           - "locked"
         php-version:
-          - "7.4"
```

Comment:
Does v1 of phpbench work on PHP 7.4?

Reply:
Yes, it supports PHP ^7.2 || ^8.0

Reply:
@t0mmy742 what problems do you mean? I think the only one is the object/resource isn't it? That's being ignored atm.

Reply:
Yes, it's about this

Reply:
So we can have both, shouldn't cause any issue on benchmarks. That's mostly out of curiosity and I'm planning to use their comparison tooling (that comes with v1.0).

I'll rebase your PR and get it merged!